api/src/database/helpers/schema/types.ts

```diff
@@ -139,4 +139,8 @@ export abstract class SchemaHelper extends DatabaseHelper {
 	formatUUID(uuid: string): string {
 		return uuid; // no-op by default
 	}
+
+	async getDatabaseSize(): Promise<number> {
+		return 0;
+	}
```

Comment:
```suggestion

	/**
	 * @returns Size of the database in bytes
	 */
	async getDatabaseSize(): Promise<number> {
		return 0;
	}
```

Adding the jsdoc in the abstract class shows it then in every place (dialects included) which is helpful imo :) 

Example:

![image](https://github.com/directus/directus/assets/14810858/0b847ae6-7a99-4e58-9d99-bd6bda93f1e0)